### DISCUSSION TITLE
header(http)

### DISCUSSION THREAD

**Discussion by solidwar** [Gozargah/Marzban#281] [Q&A]
سلام. فایل xray رو ادیت کردم و https://github.com/Gozargah/Marzban/discussions/146 این فایلی که دوستمون فرستاده بود رو جایگزین کردم اما بازم با هدر کار نمیکنه 
دانش لازم برای ساختش هم ندارم  چند بار تست کردم دیدم نشد 
فقط برای vless.vmess. tcp.ws استفاده کنم کافیه 
اگه میشه راهنمایی دقیق یا فایل xray مربوط به این رو بسازید تا ببینم بفهمم چی به چیه؟ 

و مشکل دیگه ای که دارم اینه که تو این روش لازم نیست tls فعال باشه. همین که ssl بگیری و روی آیپی دامنه خودتو قرار بدی سرعت بهتری داره تا اینکه tls فعال کنی. (تو فایل ها tls روی اینباند فعاله خودکار)

**Comment by IRANWEBFA** [comment]
```
{
      "tag": "VMESS_TCP_HTTP",
      "listen": "0.0.0.0",
      "port": 80,
      "protocol": "vmess",
      "settings": {
        "clients": [],
        "decryption": "none"
      },
      "streamSettings": {
        "network": "tcp",
        "security": "none",
        "tcpSettings": {
          "acceptProxyProtocol": false,
          "header": {
            "type": "http",
            "request": {
              "path": [
                "/"
              ]
            }
          }
        }
      }
    }
```
